tests/lib/rules/block-scope-case.test.js

```diff
@@ -22,6 +22,19 @@ validExample = formatCode(
    '}'
 );
 
+validExample2 = formatCode(
+   'switch (x) {',
+   '   case 1:',
+   '   case 2: {',
+   '      doSomethingWith2(x);',
+   '      break;',
+   '   }',
+   '   default: {',
+   '      doSomethingWithEverythingElse(x);',
+   '   }',
+   '}'
+);
+
```

Comment:
Could we add an invalid test that mirrors this without the block scope after `case 2`. And leave the block scope on `default`?  I'm sure it will pass, but I want to make sure we don't get false negatives.

Reply:
Like this?
```
switch (x) {
   case 1:
   case 2:
      doSomethingWith2(x);
      break;
   default: {
      doSomethingWithEverythingElse(x);
   }
}
```

Reply:
Yeah 👍